### PR TITLE
Feature/46 timeline frontend

### PR DESCRIPTION
**Pull request by oguzsenna**

### CODE REVIEW

File: backend/frontend/src/pages/search/StorySearch.js [hunk 1]
```diff
@@ -43,193 +43,205 @@ const StorySearch = () => {
     navigate(`/user-profile/${id}`);
   };
 
-  const handleSearch = async (e,pageNumber = 1) => {
+  const handleStoryClickWithLocation = async (locationSearch) => {
+    const locationJSON = JSON.stringify(locationSearch);
+
+
+    const url = `/timeline/${locationJSON}`;
+    console.log(url); // Add this line
```
Comment (sadikkuzu) [inline backend/frontend/src/pages/search/StorySearch.js:51]: we may comment out or remove this line

File: backend/frontend/src/pages/search/StorySearch.js [hunk 2]
```diff
@@ -43,193 +43,205 @@ const StorySearch = () => {
...
+          size: pageSize,
+          time_type: timeType,
+          time_value: JSON.stringify(timeValueObj),
+          location: JSON.stringify(locationSearch),
+          radius_diff: radiusDiff,
+          date_diff: dateDiff,
+        },
         withCredentials: true,
       });
+      console.log(locationSearch)
```
Comment (sadikkuzu) [inline backend/frontend/src/pages/search/StorySearch.js:97]: we may comment out or remove this line

File: backend/frontend/src/pages/search/Timeline.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+    handleSearch();
+  }, [locationJSON, radiusDiff]);
+
+  const handleStoryClick = async (id) => {
+    window.location.href = `/story/${id}`;
+  };
+
+  const handleGoBack = () => {
+    navigate(-1); // This will navigate back to the previous page
```
Comment (sadikkuzu) [inline backend/frontend/src/pages/search/Timeline.js:45]: Cool action 😎 👍🏼

File: backend/frontend/src/pages/search/Timeline.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                  <p className="story-author-search">by {story.author_username || 'Unknown'}</p>
+                </div>
+              </div>
+            ))}
+          </div>
+          <button onClick={handleGoBack}>Go Back</button>
+        </>
+      ) : (
+        <div>
+          <h1>Sorry, there are no stories on this location.</h1>
```
Comment (sadikkuzu) [inline backend/frontend/src/pages/search/Timeline.js:69]: Cool info 😎 👍🏼

**Review by sadikkuzu — APPROVED** [review]
Thanks ✨ 👏🏼 
I've a few review comments for console.log's; but, overall, PR is ok 👍🏼 
It's up to you to handle those lines.

**Review by sadikkuzu — CHANGES_REQUESTED** [review]
This branch is out-of-date with the base branch
Merge the latest changes from `dev` into this branch.

image

**Review by ckertam — APPROVED** [review]
I've add a time info and removed the unneccesary console logs. It's ready to merge now!

File: backend/frontend/src/pages/search/StorySearch.js (hunk 2 above)
Comment (ckertam) [reply backend/frontend/src/pages/search/StorySearch.js]: I've remove this line!

**Pull request merged**